Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -44,6 +49,15 @@ public void DispatchValidationFailures (IBusinessObjectValidationResult validati
 
       if (dispatchToValidationFailureRepository)
         DispatchToValidationFailureRepository(validationResult, bocListControl);
+
+      var control = GetControlToValidate();
+
+      _validationFailuresForBusinessObjectPropertyOfBocList = validationResult.GetValidationFailures(control, markAsHandled: true).ToArray();
```

Comment:
I don' think it is necessary to explicitly extract the validation failures here. We already put the failures into the repository, which we can then use to get the failures in `EvaluateIsValid`. (see comment below)